tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -205,32 +121,32 @@ func executeTestForFlags(flags []string, m *testing.M) (successCode int) {
 func TestMain(m *testing.M) {
```

Comment:
why are these methods still in this file

Reply:
fixed